File: src/main/java/um/haberes/report/model/haberes/core/CategoriaDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CategoriaDto {

    private Integer categoriaId;

    private String nombre = "";

    private BigDecimal basico = BigDecimal.ZERO;

    private byte docente = 0;

    private byte noDocente = 0;

    private byte liquidaPorHora = 0;

}

File: src/main/java/um/haberes/report/kotlin/dto/CategoriaDto.kt
package um.haberes.report.kotlin.dto

import java.math.BigDecimal

data class CategoriaDto(

    var categoriaId: Int? = null,
    var nombre: String = "",
    var basico: BigDecimal = BigDecimal.ZERO,
    var docente: Byte = 0,
    var noDocente: Byte = 0,
    var liquidaPorHora: Byte = 0

)
